conan/clang-tools-extra/conanfile.py

```diff
@@ -48,16 +43,15 @@ class ClangToolsExtraConan(ConanFile):
 
     def source(self):
         git_cache = os.environ.get("CONAN_LLVM_GIT_CACHE")
-        cache_hit = lambda: os.path.exists(f"{git_cache}/.git")
+        cache_hit = os.path.exists(f"{git_cache}/.git")
         cache_arg = f" --reference-if-able '{git_cache}' " if git_cache else ""
 
         if git_cache and cache_hit():
-            self.output.info(
-                f"Cache hit! Some Git objects will be loaded from '{git_cache}'."
-            )
+            self.output.info(f"Cache hit! Some Git objects will be loaded from '{git_cache}'.")
 
         self.run(
-            f"git clone {cache_arg} -b {LLVM_TAG} --single-branch https://github.com/llvm/llvm-project.git"
+            f"git clone {cache_arg} -b {LLVM_TAG} "
+            "--single-branch https://github.com/llvm/llvm-project.git"
         )
 
         if git_cache and not cache_hit():
```

Comment:
Another instance where you now need a non-function call. (I haven't scanned the file for other places needing the same change):
```suggestion
        if git_cache and not cache_hit:
```

Reply:
Fixed in dc60201d2a76.